gateway.go

```diff
@@ -0,0 +1,17 @@
+package nat
+
+import (
+	"net"
+
+	"github.com/libp2p/go-netroute"
+)
+
+func getDefaultGateway() (net.IP, error) {
+	router, err := netroute.New()
+	if err != nil {
+		return nil, err
+	}
+
+	_, ip, _, err := router.Route(net.IPv4zero)
```

Comment:
this is v4 centric. may want to try on both v4 and v6

Reply:
We currently _only_ support V4 anyways because that's what our port mapping protocols support. When we fix #16, we'll need to use IPv6.